config/env/all.js

```diff
@@ -5,5 +5,6 @@ const rootPath = path.normalize(path.join(__dirname, '/../..'));
 module.exports = {
   root: rootPath,
   port: process.env.PORT || 3000,
-  db: process.env.MONGOHQ_URL || 'mongodb://localhost:27017/cfh'
+  db: process.env.MONGOHQ_URL || 'mongodb://localhost:27017/cfh',
+  secret: 'kissaSecret'
```

Comment:
**CRITICAL - SECURITY**
This secret value should be hidden. Make use of an environment variable.